### PR TITLE
Feature/breed drop down

### PR DESCRIPTION
**Pull request by sydnerd**
### Acceptance Criteria

As a user, there is a searchable drop down menu with all breeds and a mixed breed option.

### Issues Resolved
Drop down with all breeds and a mixed breed option

### Code Reviewers
@alexmkio

### CODE REVIEW

File: src/components/app/App.js [hunk 1]
```diff
@@ -11,14 +11,20 @@ import { Event } from '../event/Event'
...
 
 export const App = () => {
   const { loading, error, data } = useQuery(QUERY_EVERYTHING)
   const [user, setUser] = useState({});
   const [dogs, setDogs] = useState([]);
   const [events, setEvents] = useState([]);
+  const [breeds, setBreeds] = useState([])
 
   useEffect(() => {
+    fetchBreeds()
```
Comment (alexmkio) [inline src/components/app/App.js:24]: Any chance you tried having this fetch inside Create component?

File: src/components/create/Create.js [hunk 2]
```diff
@@ -42,6 +42,9 @@ const Create = ({ userID }) => {
   if (loading) return 'Submitting...';
   if (error) return `Submission error! ${error.message}`;
 
+  const options = breeds.map(breed=> ({label: breed.name, value: breed.name}))
```
Comment (alexmkio) [inline src/components/create/Create.js:45]: does the returned object need to be in parenthesis? Also, add a space in front of arrow function.

File: yarn.lock [hunk 3]
```diff
@@ -1142,24 +1142,10 @@
...
-  "resolved" "https://registry.npmjs.org/@babel/runtime/-/runtime-7.15.3.tgz"
-  "version" "7.15.3"
-  dependencies:
-    "regenerator-runtime" "^0.13.4"
-
-"@babel/runtime@^7.9.2":
-  "integrity" "sha512-OvwMLqNXkCXSz1kSm58sEsNuhqOx/fKpnUnKnFB5v8uDda5bLNEHNgKPvhDN6IU0LDcnHQ90LlJ0Q6jnyBSIBA=="
-  "resolved" "https://registry.npmjs.org/@babel/runtime/-/runtime-7.15.3.tgz"
-  "version" "7.15.3"
+"@babel/runtime@^7.1.2", "@babel/runtime@^7.10.2", "@babel/runtime@^7.11.2", "@babel/runtime@^7.12.0", "@babel/runtime@^7.12.1", "@babel/runtime@^7.12.5", "@babel/runtime@^7.13.10", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.2", "@babel/runtime@^7.8.4", "@babel/runtime@^7.8.7", "@babel/runtime@^7.9.2":
```
Comment (alexmkio) [inline yarn.lock:1145]: look at all this crazy yarn.lock garbage. lol

File: src/components/create/Create.js (hunk 2 above)
Comment (sydnerd) [reply src/components/create/Create.js]: Ill fix this!

**Pull request merged**